docs/docker_run.rst

```diff
@@ -127,44 +135,47 @@ container has finished running.
 Using named pipes to stream data in and out of containers
 ---------------------------------------------------------
 
-girder_worker uses named pipes as a language agnostic way of streaming data in
+Girder Worker uses named pipes as a language agnostic way of streaming data in
 and out of docker containers. Basically a named pipe is created at a path that is
 mounted into the container. This allows the container to open that pipe for read or
-write and similarly the girder_worker infrastructure can open the pipe on the host,
-thus allow data write and read from the container.
+write and similarly the Girder Worker infrastructure can open the pipe on the host,
+thus allowing data write and read from the container.
 
-The are two utility classes using to represent as named pipe, ``NamedOutputPipe``
-and ``NamedInputPipe``
+The are two utility classes using to represent as named pipe,
+:py:class:`girder_worker.docker.transforms.NamedOutputPipe`
+and :py:class:`girder_worker.docker.transforms.NamedInputPipe`.
 
 NamedOuputPipe
 ^^^^^^^^^^^^^^
 
-This represents a named pipe that will be opened in the docker container for write.
-So is using to stream data out of a container.
+This represents a named pipe that can be opened in a docker container for write.
+Allowing data to be streamed out of a container.
```

Comment:
Sentence fragment, I think this should be after a comma rather than its own sentence